⚡ VELOCITY

thurdev opened the pull request:
Removed puppeteer which increases the velocity x10000000, add option to automatically buy items just by placing the id inside items.json, like [123123123]

![image](https://user-images.githubusercontent.com/34294813/232263295-eaba3c9e-801c-4afc-b4f4-0f5a70bba584.png)

thurdev committed:
Removed puppeteer which increases the velocity x10000000, add option to automatically buy items just by placing the id inside items.json, like [123123123]

thurdev committed:
changed log :)

facalz commented on src/types/types.d.ts:
```diff
@@ -49,21 +49,22 @@ export type MarketPlaceItemDetail = {
-  isUnder13: false;
-  name: string;
-  userId: string;
+  // displayName: string;
+  // hasVerifiedBadge: boolean;
+  // is13orOver: boolean;
+  // isAuthenticated: boolean;
+  // isPremiumUser: boolean;
+  // isUnder13: false;
+  Name: string;
+  // userId: string;
```
remove comments

facalz commented on src/index.ts:
```diff
@@ -7,21 +7,20 @@ import {
 let isRunning = false;
 const hasNoStock: number[] = [];
-
 // clear console
```
remove comment

facalz commented on yarn-error.log:
???????????????????????????????????????????????????????????????

thurdev committed:
🔥 chore(.gitignore): remove yarn.lock and add *.log to gitignore
🎨 style(index.ts): remove unnecessary comments and format code
🎨 style(types.d.ts): remove commented out code and format file

facalz committed:
Delete yarn-error.log
